lib/db.js

```diff
@@ -78,7 +78,16 @@ module.exports = (
   }
 
   DB.connect = function (options) {
-    return P.resolve(new DB(options))
+    const db = new DB(options)
+    if (! db.redis) {
+      return P.resolve(db)
+    }
+
+    return new P(resolve => {
+      db.redis.on('ready', () => {
```

Comment:
When testing this locally (forgot to start redis) auth server returns `Unexpected Error` for all requests. Should it do this, or should we disable redis until it gets reconnected?

Reply:
Good question! Not sure, having a think about it...